go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/mitchellh/cli
 
+go 1.14
```

Comment:
I believe the last 2 or 3 versions of Go will automatically add this directive to the `go.mod` file. Adding it will make it slightly more convenient for contributors so they do not have to revert the changes made by the toolchain every time they run tests.

I can change this to an older version if that is desirable.

Reply:
CI was failing on Go1.11 due to go modules being enabled, change to `go1.11`